src/sqlalchemy_declarative_extensions/dialects/postgresql/trigger.py

```diff
@@ -133,6 +134,9 @@ def for_each_statement(self):
     def when(self, condition: str):
         return replace(self, condition=condition)
 
+    def with_arguments(self, arguments: list[str]):
```

Comment:
For a builder method like this, what do you think of:

```suggestion
    def with_arguments(self, *arguments: str):
```

two fewer characters at the call site